src/openrct2/rct12/RCT12.cpp

```diff
@@ -750,6 +750,14 @@ static constexpr std::string_view _musicStyles[] = {
     "rct2.music.pirate",
     "rct2.music.rock3",
     "rct2.music.candy",
+    "openrct2.music.galaxy",
```

Comment:
This will only add it for sc6/sc4 scenarios what about .park scenarios?

Reply:
I'll look into it! Might be smart to do it there, too, tho I think as long as it's part of the basic selection when creating a scenario it should be fine, too.

Reply:
I have implemented it to be part of any sc6/sc4 that gets opened, and it is now part of the default selection when creating a new .park scenario. It does not alter existing .park scenarios. For official scenarios the list should be updated in the scenario itself.